src/Integration.Vsix.UnitTests/ErrorList/IssuesSnapshotFactoryTests.cs

```diff
@@ -32,12 +32,20 @@ namespace SonarLint.VisualStudio.Integration.UnitTests.ErrorList
     [TestClass]
     public class IssuesSnapshotFactoryTests
     {
+        [TestMethod]
+        public void Create_CurrentSnapshotIsEmpty()
+        {
+            var testSubject = new IssuesSnapshotFactory("my file");
+            testSubject.CurrentSnapshot.GetType().Name.Should().Be("EmptyIssuesSnapshot");
```

Comment:
I made the nested class `private` so the test code can't see it at all -> need to check the type by name.